working/1426-extension-types/feature-specification-views.md

```diff
@@ -546,48 +724,127 @@ or
 is used to invoke these constructors, and the type of such an expression is
 <code>V<T<sub>1</sub>, .. T<sub>k</sub>></code>.
 
-During static analysis of the body of a view constructor of a view which is
-not closed, the return type is considered to be the view type declared by
-the enclosing declaration.
-
-*This means that the constructor can return an expression whose static type
-is the on-type, as well as an expression whose static type is the view
-type.*
-
-During static analysis of the body of a view constructor of a view which is
-closed, the return type is considered to be the on-type of the enclosing
+During static analysis of the body of a view constructor, the return
+type is considered to be the view type declared by the enclosing
 declaration.
 
-*So these constuctors can only return an expression of the on-type, not an
-expression of the view type, but an explicit cast to the on-type can be
-used if needed.*
-
-It is a compile-time error if it is possible to reach the end of a view
-constructor without returning anything. *Even in the case where the on-type
-is nullable and the intended representation is the null object, an explicit
-`return null;` is required.*
-
-Let `V` be a view declaration. It is an error to declare a member in
-`V` which is also a member of `Object`.
-
-*This is because the members of `Object` are by default shown, as
-specified below in the section about the show/hide part. It is possible to
-use `hide` to omit some or all of these members, in which case it is
-possible to declare members in `V` with those names.*
-
-
-### Allow instance member access using `show` and `hide`
-
-This section specifies the effect of including a non-empty
-`<viewShowHidePart>` in a view declaration.
-
-*The show/hide part provides access to a subset of the members of the
-interface of the on-type. For instance, if the intended purpose of the
-view type is to maintain a certain set of invariants about the state
-of the on-type instance, it is no problem to let clients invoke any methods
-that do not change the state. We could write forwarding members in the
-view body to enable those methods, but using show/hide can have the
-same effect, and it is much more concise and convenient.*
+*This means that the constructor can return an expression whose static type
```

Comment:
I'm not sure how I feel about this approach, relative to just saying that you need to initialize the unique field as normal, unless it's a factory.